Wire up support for converting integrations-next configs to flow. (#5973)
grafana/agent: docs/sources/flow/reference/cli/convert.md

Using the `--source-format=static` will convert the source configuration from a
[Grafana Agent Static]({{< relref "../../../static" >}}) configuration to a {{< param "PRODUCT_NAME" >}} configuration.

Include `--extra-args` for passing additional command line flags from the original format.

> For example, `--extra-args="-enable-features=integrations-next"` will convert a static

Reviewer: ```suggestion
For example, `--extra-args="-enable-features=integrations-next"` will convert a Grafana Agent Static
```

[integrations-next]({{< relref "../../../static/configuration/integrations/integrations-next/" >}})
configuration to a {{< param "PRODUCT_NAME" >}} configuration. You can also

> expand env vars by including the `--extra-args="-config.expand-env"`. Multiple command line

Reviewer: ```suggestion
expand environment variables with `--extra-args="-config.expand-env"`. You can combine multiple command line
```

> flags can be combined with a space between, such as `--extra-args="-enable-features=integrations-next -config.expand-env"`.

Reviewer: ```suggestion
flags with a space between each flag, for example `--extra-args="-enable-features=integrations-next -config.expand-env"`.
```
